src/commands/synthetics/README.md

```diff
@@ -213,3 +214,22 @@ Took 11546ms
     ⎋  total duration: 28.9 ms - result url: https://app.datadoghq.com/synthetics/details/abc-def-ghi?resultId=123456789123456789
     ✓ GET - https://testing.website
 ```
+
+### Reporters
+
+We currently only use a default reporter writing in stdout.
+
+Reporters can hook themselves into the `MainReporter` of the command.
+
+#### Available hooks
+
+| Hook name     | Parameters                                                                              | Description                                                     |
+| :------------ | :-------------------------------------------------------------------------------------- | :-------------------------------------------------------------- |
+| `error`       | `(error: string)`                                                                       | called whenever an error occurs.                                |
+| `initErrors`  | `(errors: string[])`                                                                    | called whenever an error occurs during the tests parsing phase. |
+| `log`         | `(log: string)`                                                                         | called for logging.                                             |
+| `runEnd`      | `(summary: Summary)`                                                                    | called at the end of the run.                                   |
+| `start`       | `(timings: {startTime: number})`                                                        | called at the begining of the run.                              |
```

Comment:
It seems to be called at the beginning of the report, could you update the description so it is clearer? (or rename it to `reportStart`?)